tests/contrib/anthropic/test_anthropic_patch.py

```diff
@@ -0,0 +1,21 @@
+from ddtrace.contrib.anthropic import get_version
+from ddtrace.contrib.anthropic import patch
+from ddtrace.contrib.anthropic import unpatch
+from tests.contrib.patch import PatchTestCase
+
+
+class TestAnthropicPatch(PatchTestCase.Base):
+    __integration_name__ = "anthropic"
+    __module_name__ = "anthropic"
+    __patch_func__ = patch
+    __unpatch_func__ = unpatch
+    __get_version__ = get_version
+
+    def assert_module_patched(self, anthropic):
+        self.assert_wrapped(anthropic.resources.messages.Messages.create)
```

Comment:
We want to assert that `AsyncMessages` is also wrapped/not/double wrapped

Reply:
done